releases/v2.0-beta.20180305.md

```diff
@@ -0,0 +1,406 @@
+
+### Enterprise Edition Changes
+
+- Bulk IO rate limiting is now done per-store rather than per-node. [#22729][#22729]
+- Revision history backup and restore no longer considered experimental. [#22679][#22679]
+- Revision history backup/restore now handles schema changes. [#21717][#21717]
+- CockroachDB now checks that backup actually contains requested restore time. [#22659][#22659]
+- Improved `BACKUP`'s handling of tables after `TRUNCATE` [#21895][#21895]
+- Removed `into_db` option in `IMPORT`. The database is now specified as part of the table name. [#21813][#21813]
+- Ensured that only the backups created by the same cluster can be used in incremental backups. [#22474][#22474]
+- Avoided extra internal copying of files during `RESTORE`/`IMPORT`. [#22281][#22281]
+- [Sequences](../v2.0/create-sequence.html) are now supported in enterprise [`BACKUP`](../v2.0/backup.html)/[`RESTORE`](../v2.0/restore.html) jobs.
+
+    This changes how sequences are stored in the key-value storage layer, so existing sequences must be dropped and recreated. Since a sequence cannot be dropped while it is being used in a column's `DEFAULT` expression, those expressions must be dropped before the sequence is dropped, and recreated after the sequence is recreated. The `setval` function can be used to set the value of a sequence to what it was previously. [#21684][#21684]
+
+- Added a geographical map to the homepage of the Admin UI enterprise version, showing the location of nodes and localities in the cluster. The map is annotated with several top-level metrics: storage capacity used, queries per second, and current CPU usage, as well as the liveness status of nodes in the cluster. [#22763][#22763]
+
+### SQL Language Changes
+
+- The type determined for constant `NULL` expressions is renamed to `unknown` for better compatibility with PostgreSQL. [#23150][#23150]
+- Deleting multiple rows at once now consumes less memory. [#23013][#23013]
+- Attempts to modify virtual schemas with DDL statements now fail with a clearer error message. [#23041][#23041]
+- The new statement `SHOW SCHEMAS` reveals which are the valid virtual schemas next to the physical schema `public`. [#23041][#23041]
+- CockroachDB now recognizes the special syntax `SET SCHEMA <name>` as an alias for `SET search_path = <name>` for better compatibility with PostgreSQL. [#23041][#23041]
+- `current_role` is recognized as an alias for `current_user` for better compatibility with PostgreSQL. [#23041][#23041]
+- The special identifier `current_catalog` is supported as alias for `current_database()` for better compatibility with PostgreSQL. [#23041][#23041]
+- CockroachDB now returns the correct error code for division by zero. [#22948][#22948]
+- The GC of table data after a `DROP TABLE` now respects changes to the GC TTL interval specified in the zone config [#22903][#22903]
+- The full names of tables/view/sequences are now properly logged in the system event log. [#22848][#22848]
```

Comment:
Are we announcing the "system even log" here, or just announcing a fix of some bug? If the latter, let's move under bug fixes. If the former, let's say "are logged in the new system event log".

Reply:
We mention "auditing" and "SQL execution log" in a few bullets below. I suspect we're talking about the same thing using different terms. If so, we should consolidate or, at the least, use the same terminology. cc @knz for his insights.

Reply:
No the audit log and the execution log are two distinct, unrelated features.